app/views/admin/shared/_sidenav_billing.html.haml

```diff
@@ -11,8 +11,9 @@
     = tab t("facilities.disputed_orders.head"), facility_disputed_orders_path, (sidenav_tab == "disputed_orders")
   %li.divider
   = tab t('facility_accounts.accounts_receivable.head'), facility_accounts_receivable_path, (sidenav_tab == 'accounts_receivable')
-  %li.divider
-  = tab Account.model_name.human.pluralize, facility_accounts_path, (sidenav_tab == 'accounts')
+  - if current_user.billing_administrator?
```

Comment:
I've got mixed feeling about this (and the same change in `vendor/engines/c2po/app/views/admin/shared/_sidenav_billing.html.haml`). I would have liked to make this an ability check against `Account` but too many managerial roles have `:manage` ability on `Account` and we need this link to show up for Billing Administrators only.

There's also the issue that once clicking through, it takes the user to an `admin_users` tab context rather than `admin_billing`.